src/CMakeLists.txt

```diff
@@ -54,6 +60,13 @@ qt_add_qml_module(${PROJECT_NAME}
         QtQuick.Controls.Basic
 )
 
+list(APPEND QML_DIRS "/usr/lib64/qt6/qml")
```

Comment:
It's not good to add a fixed path. You can try to use `Qt6_DIR` to make it relocatable.

Reply:
Removed this line